packages/logs/src/domain/logsCollection/console/consoleCollection.ts

```diff
@@ -31,7 +31,9 @@ export function startConsoleCollection(configuration: LogsConfiguration, lifeCyc
           log.api === ConsoleApiName.error
             ? {
                 origin: ErrorSource.CONSOLE, // Todo: Remove in the next major release
-                stack: log.stack,
+                kind: log.kind,
+                stack: log.stack || NO_ERROR_STACK_PRESENT_MESSAGE,
+                message: log.errorMessage || `${PROVIDED_ERROR_MESSAGE_PREFIX} ${log.message}`,
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏same thing here, we might be able to use the same serialization logic